### ISSUE TITLE
Option to turn gray icon square on/off

### ISSUE THREAD

**Issue by BeckysFootSlave** [CTCaer/hekate#739]
A friend and myself have a project together!
Started with a vertical nxtheme using a switch Cardrige as game icons!

He suggested to also use that Cardrige for hekate 😉
Here is a preview of how it looks:
![IMG_7623](https://user-images.githubusercontent.com/13203024/164166597-2d72b839-b3b2-4964-80a2-73a5c5716931.jpg)

Now we ask you if we can get rid of the gray square or a on/off function because the icons have correct alpha channel settings to just show the Cardrige!

Another idea I had in mind was a new icon size for this icons 😬
Still 192 in height but just 128 in width to get 5 icons aligned in one row!
But that just came to my mind!
I can't code and that is the reason I never know how hard it really is to implement such things!

I thank you so much for your time and one of the greatest (of not the greatest) bootloader I have ever seen!

**Comment by Dankal87** [comment]
Le dot

**Comment by urherenow** [comment]
Couple of suggestions for your icons: OFW is only done with the reboot -> OFW option, so an icon for that is not useful. It should be labeled "stock" instead of OFW, because it's not exactly the same thing. Atmosphere  (payload=bootloader/payloads/fusee.bin) is emuEMMC, if you have one and it is active, yet will launch CFW emmc if you don't have one... however, with fss0, you can choose to launch CFW in emmc OR emuEMMC, so there should be 2 different icons.

Have you posted these icons to gbatemp? Please do so, if not.

**Comment by BeckysFootSlave** [comment]
> Couple of suggestions for your icons: OFW is only done with the reboot -> OFW option, so an icon for that is not useful. It should be labeled "stock" instead of OFW, because it's not exactly the same thing. Atmosphere (payload=bootloader/payloads/fusee.bin) is emuEMMC, if you have one and it is active, yet will launch CFW emmc if you don't have one... however, with fss0, you can choose to launch CFW in emmc OR emuEMMC, so there should be 2 different icons.
> 
> Have you posted these icons to gbatemp? Please do so, if not.

That is a picture of my friend's hekate!
But thank you for mentioning!
Tbh I have also named it OFW (sysMMC) = (Stock) 😬 but I use two different for atmosphere!
![OFW sysMMC](https://user-images.githubusercontent.com/13203024/164225325-cdacd798-5033-40c6-a629-66ac96718083.png)
![Atmosphere sysMMC](https://user-images.githubusercontent.com/13203024/164225367-2efc8a66-a010-42a6-9f2e-53fab39b60b0.png)
![Atmosphere emuMMC](https://user-images.githubusercontent.com/13203024/164225396-a1aa2cd1-a2fb-409c-a661-af40bdf4ffe9.png)
![SwichDroid](https://user-images.githubusercontent.com/13203024/164225420-8f77ebd6-158d-4ddd-bc56-58cd198426ca.png)
![Ubuntu L4T](https://user-images.githubusercontent.com/13203024/164225444-dd064d37-a5c5-4b5b-878c-7e3a48b4bfce.png)
![LAKKA](https://user-images.githubusercontent.com/13203024/164225464-a8f9f67c-20a4-4ebf-858c-fbb258e3988a.png)

These are mine and I also use a different background (but the same frame (frame is made by me)
I will tell him that you said to bring them to Gbatemp! 😉

**Comment by BeckysFootSlave** [comment]
Looks really good without border! 
Next test is with a 2x5 grid! 
![Picsart_22-04-30_19-04-31-669](https://user-images.githubusercontent.com/13203024/166115352-53640be8-6a26-4513-bc42-8f709cb278c1.png)

**Comment by BeckysFootSlave** [comment]
Also the 5x2 grid works like a charm!
No problems encountered while using it!
Just awesome... Speechless! 

![nyx20220507_182027](https://user-images.githubusercontent.com/13203024/167546894-2066f6a9-762f-40a2-8bf6-75257198a466.png)

**Comment by impeeza** [comment]
Hello there, I don't know if comment here or is better to open a new isse.

on Hekate 5.8.0 if you use a file icon with "HUE" format, it is displayed well and with the color of interface but if THE SAME file is renamed `*_nobox.bmp` then the hue color isn't used the icon is show without the gray background but white.

![nyx20220721_015750](https://user-images.githubusercontent.com/41979604/180149872-505a53cd-7226-4bf5-9790-23dd2b8f389e.png)

on this example all icons are the special ones with "Hue" format, but the second one is named `Payload_nobox.bmp` I don't know if is the wanted behavior or a bug introduced with the new functionalities.

**Comment by BeckysFootSlave** [comment]
I don't think that you can mix them up!

And if you can do that = just "nobox" doesn't know that the image is hue and should change the color!
It would be possible if the ending is = icon_hue_nobox.bmp
(don't know if that works?!

But just then nyx would know it is a hue icon with color changing and without the box!

**Comment by impeeza** [comment]
Yep, in fact the file currently is called: SD:\bootloader\res\payload_hue_nobox.bmp

**Comment by impeeza** [comment]
Yep, in fact the file currently is called: SD:\bootloader\res\payload_hue_nobox.bmp

**Comment by alimazharali** [comment]
> Looks really good without border! Next test is with a 2x5 grid! ![Picsart_22-04-30_19-04-31-669](https://user-images.githubusercontent.com/13203024/166115352-53640be8-6a26-4513-bc42-8f709cb278c1.png)

please can you give a link to your icons or theme ?

**Comment by BeckysFootSlave** [comment]
@alimazharali
It's all selfmade 😘
I can upload them if you want and give you a link! 👍

**Comment by alimazharali** [comment]
> @alimazharali It's all selfmade 😘 I can upload them if you want and give you a link! 👍

Excellent work ,.,., most beautiful icons i ever see ,.,.
link plssssssssssssssssss god bless you 
waiting :)

**Comment by alimazharali** [comment]
> @alimazharali It's all selfmade 😘 I can upload them if you want and give you a link! 👍

plssss send me all the icons if you can or full theme of hakete as you shown i the thread i am waiting desperately,.,. 
my email is 
alimazharali@yahoo.com

**Comment by Masamune3210** [comment]
Be careful about just posting your email in plain text. Bots crawl the internet looking for them so that they can be added to spam lists, not to mention the fact that literally anyone who looks at the comment chain or is subscribed to it has your email now

**Comment by BeckysFootSlave** [comment]
Yeah that's right what @Masamune3210 says!

I'll give link to hoster or github link

**Comment by alimazharali** [comment]
> Yeah that's right what @Masamune3210 says!
> 
> I'll give link to hoster or github link

pls notify here ,,.,. i will wait ,.,.
millions of thanks ,.,. all hekate icons pls and more if you have

**Comment by impeeza** [comment]
Hello there,  all of you if you go to GBATemp.net you will find a thread about how to create that icons, the author has shared them on the thread and you will find also the new set of naming needed for they to work.

**Comment by BeckysFootSlave** [comment]
Here the background with and without the hands and all my icons!
I'm going to make new ones and can also post them here!

[bg_and_icons.zip](https://github.com/CTCaer/hekate/files/11897442/bg_and_icons.zip)

**Comment by Jbb08** [comment]
@impeeza any chance you can point me to your green icons, I really want my icons to match the main Hekate theme, but for the life of me I can’t seem to find any, yours are the closest I can find, sorry for messaging on a closed threat :-)

**Comment by urherenow** [comment]
GitHub isn’t a support forum. If you don’t have any kind of icon entry in hekate_ipl.ini, then you’re already using “the main Hekate theme”.

**Comment by impeeza** [comment]
@Jbb08 They are not green they are «Hue» icons, the green color is the color set on NYX,  if you search me on GBATemp.net I will try to help you.
